frontend/src/features/book/BookSection.tsx

```diff
@@ -0,0 +1,36 @@
+import { useGetBook } from "@/apis/BookAPI";
+import BookDetailSection from "@/features/book/BookDetailSection";
+import BookListSection from "@/features/book/BookListSection";
+import { css } from "@emotion/react";
+import { useState } from "react";
+
+const BookSection = () => {
+  const [activeTab, setActiveTab] = useState<string>("신규 예약");
+  const [activeBookId, setActiveBookId] = useState<number | null>(null);
+
+  const { data, refetch } = useGetBook(activeTab);
+  const activeBookData = data?.find(
+    (book) => book.id === (activeBookId ? activeBookId : data[0].id),
```

Comment:
null일 때 뒤에 값을 쓰시고 싶다면 해당 코드도 추천합니다
```suggestion
    (book) => book.id === (activeBookId ?? data?.[0]?.id),
```